module/input/SensorFilter/src/SensorFilter.cpp

```diff
@@ -475,6 +475,17 @@ namespace module::input {
                             sensors->gyroscope = input.gyroscope.cast<double>();
                         }
 
+                        // Add gyro and acc graphs if in debug
+                        if (log_level <= NUClear::DEBUG) {
+                            emit(graph("Gyro x-axis", sensors->gyroscope.x()));
+                            emit(graph("Gyro y-axis", sensors->gyroscope.y()));
+                            emit(graph("Gyro z-axis", sensors->gyroscope.z()));
```

Comment:
If you emit a list of 2-3 items at the same time they'll be combined and labelled x, y, z automatically when plotting. Will include this in the PlotJuggler README

```suggestion
                            emit(graph("Gyro", sensors->gyroscope.x(), sensors->gyroscope.y(), sensors->gyroscope.z()));
```

Same can be used for Accelerometer below